modules/react/layout/stories/Stack.stories.mdx

```diff
@@ -12,6 +12,13 @@ import {VStackCards} from './examples/Stack/VStackCards';
 
 <Meta title="Components/Containers/Stack" component={Stack} />
 
+### ⚠️ Stack has been soft-deprecated in v8 and will be removed in v9 ⚠️
+
+- If you would like resolve this now, run this codemod: `npx @workday/canvas-kit-codemod v8 [path]`
+- You can learn more about the codemod package
+  [here](https://github.com/Workday/canvas-kit/tree/master/modules/codemod)
+- There is more context [here]() as to why this decision was made
```

Comment:
Do we mention anything in the v8 upgrade guide?

Reply:
I had to adjust this and updated the codemod command. 

This has been updated along with a link to link back to the original ticket for explanation